Grab environment from .env file

kevinemoore opened the pull request:
Add dotenv to get environment from file.

Commit:
Grab environment from .env file

Add dotenv to get environment from file.

drernie commented:
Ah, right. I think I didn't use it because of some weird interaction with aider, but that's probably cleaner.

Commit:
Merge branch 'main' into fix-env

Commit:
Add glue permissions

Adding glue:GetPartitions and extending the lambda timeout.

Commit:
Add pagination to get all tables in the database

Adding pagination to the Glue API call that fetches the list of tables from the database. The API call will only return 100 tables at a time.

Commit:
Fix tests

Commit:
Update DuckDBJoin.ipynb
